ingenialink/ethercat/servo.py

```diff
@@ -251,10 +253,21 @@ def _handle_sdo_exception(
                 f"Error {operation_msg.value} {reg.identifier}. Reason: {exception}"
             ) from exception
         elif isinstance(exception, pysoem.WkcError):
-            if exception.wkc == self.NOFRAME_WORKING_COUNTER:
-                raise ILIOError(f"Error {operation_msg.value} data: No frame.") from exception
-            if exception.wkc == self.TIMEOUT_WORKING_COUNTER:
-                raise ILTimeoutError(f"Timeout {operation_msg.value} data.") from exception
+            default_error_msg = f"Error {operation_msg.value} data"
+            wkc_exceptions: Dict[int, Exception] = {
```

Comment:
Must you use the generic Exception type? I think you could use:
`Union[ILIOError, ILTimeoutError]`

Reply:
Done